lib/vectory/svg.rb

```diff
@@ -26,7 +26,7 @@ def self.from_node(node)
     end
 
     def content
-      @document&.root&.to_xml || @content
+      @document&.root&.to_xml || @content&.sub(/^\s*<\?xml[^>]*>/m, "")
```

Comment:
Can we use an XML parser to remove this instead of using a RegExp?

Reply:
Yes 👍

Reply:
This is now addressed in [`Svg#content=`](https://github.com/metanorma/vectory/pull/38/files#diff-6aae15c6f2871104d003a3296fc7365bc28983a4709a87cbba026f1d13056850R56)